### PR TITLE
Twisted deps

### PR DESCRIPTION
**Pull request by pyhalov**

### CODE REVIEW

**Comment by pyhalov** [comment]
Twisted-web2 doesn't work with latest twisted and seems to be abandoned.
Coherence is also seems to be more dead than alive.

File: components/meta-packages/history/history [hunk 1]
```diff
@@ -880,6 +880,9 @@ library/python/python-gnome-extras-27@2.25.3,5.11-2018.0.0.5
 library/python/python-gnome-extras@2.25.3,5.11-2018.0.0.5
 library/python-2/argparse@1.2.1,5.11-2016.0.1.0
 library/python-2/argparse-26@1.2.1,5.11-2016.0.1.0
+library/python-2/coherence-26@0.6.6.2-2016.0.1.2
```
Comment (Mno-hime) [inline components/meta-packages/history/history:883]: The latest one I see is `FMRI: pkg://openindiana.org/library/python-2/coherence-26@0.6.6.2-2016.0.1.2:20180211T133129Z`, should this be **2016.0.1.3**?

File: components/meta-packages/history/history [hunk 2]
```diff
@@ -912,13 +915,20 @@ library/python-2/python-notify-24@0.5.11,5.11-2015.0.2.0
 library/python-2/python-pigment-26@0.5.11,5.11-2015.0.2.0
 library/python-2/python-twisted-24@0.5.11,5.11-2015.0.2.0
 library/python-2/python-twisted-web2-24@0.5.11,5.11-2015.0.2.0
+library/python-2/python-twisted-web2-26@8.1.0,5.11-2016.0.1.0
```
Comment (Mno-hime) [inline components/meta-packages/history/history:918]: **2016.0.1.1**?

**Review by Mno-hime — COMMENTED** [review]
Except for those two history changes, LGTM (builts and installs fine).

File: components/meta-packages/history/history (hunk 1 above)
Comment (pyhalov) [reply components/meta-packages/history/history:883]: No, as we just move definition from coherence history file to global history file (the package is already obsolete).

File: components/meta-packages/history/history (hunk 2 above)
Comment (pyhalov) [reply components/meta-packages/history/history:918]: The same here.

**Pull request merged**